core-bundle/src/Controller/ContentElement/AccordionController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Controller\ContentElement;
+
+use Contao\ContentModel;
+use Contao\CoreBundle\DependencyInjection\Attribute\AsContentElement;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\Twig\FragmentTemplate;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+#[AsContentElement(category: 'miscellaneous', nestedFragments: true)]
+class AccordionController extends AbstractContentElementController
+{
+    public function __construct(private readonly ContaoFramework $framework)
+    {
+    }
+
+    protected function getResponse(FragmentTemplate $template, ContentModel $model, Request $request): Response
+    {
+        $adapter = $this->framework->getAdapter(ContentModel::class);
+        $elements = [];
+
+        foreach ($template->get('nested_fragments') as $reference) {
+            $model = $adapter->findByPk($reference->getContentModel());
```

Comment:
```suggestion
            $model = $reference->getContentModel() instanceof ContentModel ? $reference->getContentModel() : $adapter->findByPk($reference->getContentModel());
```
`getContentModel()` could already return the model here I think.

Reply:
Fixed in 96d32378ce7ef121286021c5261527c1b7a87945.